runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/harness/StreamingWorkerStatusPages.java

```diff
@@ -150,6 +157,16 @@ private void addStreamingEngineStatusPages() {
     statusPages.addCapturePage(Preconditions.checkNotNull(channelzServlet));
     statusPages.addStatusDataProvider(
         "streaming", "Streaming RPCs", Preconditions.checkNotNull(windmillStreamFactory));
+    statusPages.addStatusDataProvider(
+        "jobSettings",
+        "User Worker Job Settings",
+        writer -> {
+          if (seConfig.get() == null) {
```

Comment:
seems safer to store result of get() on stack instead of calling twice since it can change

Reply:
Done.